packages/core/src/components/TextArea/TextArea.types.ts

```diff
@@ -50,4 +50,12 @@ export interface TextAreaProps extends TextAreaNativeInputProps, VibeComponentPr
    * Placeholder text to display when the textarea is empty.
    */
   placeholder?: string;
+  /**
+   * The allowed number of characters.
+   */
+  maxLength?: number;
+  /**
+   * If true, the textarea will allow the user to exceed the character limit.
+   */
+  allowExceedingLimit?: boolean;
```

Comment:
Not sure we need this param.
If you set a limit, why would you want to allow it to be exceeded?

Reply:
Per our conversation this morning, we want to allow the limit to be exceeded to enable a user to type out their original intended text such that they can more easily reduce it afterwards.

Followed the designs and AFAIK design is aligned on this 👍